packages/react-shortcuts/src/ShortcutManager/ShortcutManager.tsx

```diff
@@ -67,7 +67,10 @@ export default class ShortcutManager {
     }
 
     function keyGroupIsHeld(keyGroup: ModifierKey[]) {
-      return keyGroup.every((key: ModifierKey) => event.getModifierState(key));
+      return keyGroup.every(
+        (key: ModifierKey) =>
+          event.getModifierState && event.getModifierState(key),
```

Comment:
Since this is a bug fix, can you add a changelog to [this file](https://github.com/Shopify/quilt/blob/master/packages/react-shortcuts/CHANGELOG.md) and a test [here](https://github.com/Shopify/quilt/blob/8a9c68de5686d5f57827c7c3da7cfe375f58e837/packages/react-shortcuts/src/tests/ShortcutManager.test.tsx#L299)? I think this test should cover it:
```ts
    it('doesn’t fire on non-keyboard events', () => {
      const matchSpy = jest.fn();
      const heldGroup: ModifierKey[] = ['Control', 'Shift'];
      const heldToCheck: HeldKey = [[...heldGroup], ['Alt', 'Meta']];

      mount(
        <ShortcutProvider>
          <Shortcut held={heldToCheck} ordered={['/']} onMatch={matchSpy} />
        </ShortcutProvider>,
      );

      document.dispatchEvent(new UIEvent('keydown'));

      expect(matchSpy).not.toHaveBeenCalled();
    });
```

Reply:
Updated test case and change log.